### PR TITLE
ENH: updated pcds to 3.3.2

### PR DESCRIPTION
**Pull request by ZLLentz**
Let's let travis do its thing

### CODE REVIEW

**Comment by ZLLentz** [comment]
`hutch-python` tests are failing due to assumptions about `lightpath`. I did not remember that it used lightpath.
Environment build fails outright because it took more than 10 minutes to do the conda solve.

**Comment by ZLLentz** [comment]
Now we're getting the following for the py3.6 newest tags build:
`*** picosat: out of memory in 'resize'`
This does not happen locally for me.
In the past, I see issues on google related to using a 32-bin python (irrelevant here) and on specific versions of conda having an issue.
I will at least try pulling back to previous conda versions, despite the conda version in use here working well locally for me.

**Comment by ZLLentz** [comment]
This did not help. I can try more versions, but I may just opt to abandon those extra builds for now.
For some reason, it seems like the conda solve is using crazy amounts of memory.

**Comment by ZLLentz** [comment]
I've settled on the following explanation:
Something has changed in the available conda channels or on the travis workers and now the solve takes too much RAM and cannot complete.
For now I'll just make the offending builds allowed to fail.

File: envs/pcds/env.yaml [hunk 1]
```diff
@@ -40,10 +41,8 @@ dependencies:
   - cairo=1.16.0
   - caproto=0.5.1
```
Comment (klauer) [inline envs/pcds/env.yaml:42]: 0.5.2 would be nice, but could wait for the next release

Comment (ZLLentz) [reply envs/pcds/env.yaml:42]: 0.5.2 is not currently on conda-forge

Comment (klauer) [reply envs/pcds/env.yaml:42]: ... bumping PR at conda-forge

**Pull request merged**